index.html

```diff
@@ -0,0 +1,206 @@
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Securam Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+
+<section class="home-page">
+  <div class="title">
+    <h1 class="intro-headline">"Hello!! It's Afrobeat to the world"</h1>
+    <h1 class="main-headline" >NIGERIA MUSIC SUMMIT 2021</h1>
+    <div class="theme">
+      <p>
+        Celebration of Nigerian hiphop miusic, its global reception, and challenges. creating a positive change in Nigerian music industry.
+      </p>
+      </div>
+      <div class="date">
+       <h2> 2021.12.16(THU) ~ 17(FRI) </h2>
+       <p>@ Eko Hotels & Suites <br> 1415 Adetokunbo Ademola Street, <br>Victoria Island Lagos </p>
+      </div>
+  </div>
+</section>
+
+<!-- About section -->
+
+<section class="program">
+  <div class="main-program">
+    <h3>Main Program</h3>
+    <hr>
+  </div>
+  <div class="program-list">
+
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program1.png" alt="">
+    </div>
+      <h3>Lecture</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+  
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program2.png" alt="">
+    </div>
+      <h3>MUSIC</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+  
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program3.png" alt="">
+    </div>
+      <h3>Forum</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program4.png" alt="">
+    </div>
+      <h3>Wrokshop</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program5.png" alt="">
+    </div>
+      <h3>Q&A</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+
+
+    </div>
+
+  <button class="join" type="button"> Join the summit</button>
+</section>
+
+ <div class="section-speakers-heading" id="featured">
+      <h2>Featured Speakers</h2>
+      <hr />
+      <div class="featured-speakers" id="speakers"></div>
```

Comment:
- [ ] Please match the font and colors based on the given design. Also there should be 6 featured speakers instead of 8

Given design:
![Screenshot from 2021-11-18 19-19-57](https://user-images.githubusercontent.com/62705947/142474042-6adfc24a-bb19-46b9-bf5d-3f0188738564.png)
Yours:

![Screenshot from 2021-11-18 19-20-05](https://user-images.githubusercontent.com/62705947/142474081-78b2c321-924d-47eb-be5e-072e35f038e2.png)

Reply:
This is not well solved. Please take a look at the original design to implement it.

- You can see the image how it looks and the design on the image if it matches with yours.

-> Compare the original design with yours

![html12](https://user-images.githubusercontent.com/55635977/142590417-74b3a247-3027-40b0-aa63-e23235eb1997.png)

You can even work on the mobile section to match the given design.

![html14](https://user-images.githubusercontent.com/55635977/142591408-3b675caf-6348-4e6e-b4a9-24ef229c4672.png)